packages/bridge/src/bridge.ts

```diff
@@ -39,8 +39,15 @@ const webBridge: WebBridge | undefined = IS_BROWSER_CLIENT
 export function createCondoBridge (): CondoBridge {
     const subscribers: Array<CondoBridgeSubscriptionListener> = []
 
+    /**
+     * Send params to container
+     * @param method handler method
+     * @param params send params. Should not be sensitive data!
+     */
     function send<K extends AnyRequestMethodName> (method: K, params?: RequestParams<K> & RequestId) {
         if (webBridge && typeof webBridge.postMessage === 'function') {
+            // used for sending render info to containers only
```

Comment:
What is render info? Do you know, what bridge is about?)

Reply:
I'll remove comments to let you put proper description by yourself, since I'm not deep in purposes of bridge & UI packages